sklearn/decomposition/kernel_pca.py

```diff
@@ -35,8 +35,8 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         Degree for poly kernels. Ignored by other kernels.
 
     gamma : float, default=1/n_features
-        Kernel coefficient for rbf and poly kernels. Ignored by other
-        kernels.
+        Kernel coefficient for rbf, poly kernels, sigmoid, laplacian and chi2.
```

Comment:
Kernel coefficient for rbf, poly, sigmoid, laplacian and chi2 kernels.

I think this might need to be fixed in many places. Do a `git grep "gamma :"`